docs/governance/on-chain-process.md

```diff
@@ -44,29 +42,23 @@ If the proposal has been approved, provided the proposal’s feasibility, the xE
+You will then create the on-chain proposal: by providing the hash of the proposal text and depositing 10B MEX. 
 
 [comment]: # (mx-context-auto)
 
 ### About the Deposit
 
-As explained in the previous paragraph, before a governance proposal enters the voting period (i.e., for a proposal to be voted on), an amount of XMEX whose total Energy is at least 1.5 trillion must be accurately recorded. As an xEIP creator, you will deposit these tokens in the "Governance Depositor" smart contract.
-
-In case you have not enough funds to cover the entire deposit, the "Governance Depositor" in fact allows you to ask friends and larger Energy holders to top up the remaining XMEX tokens until it reaches the deposit threshold. This deposit threshold must be attained within a maximum of 10 days, otherwise all users will be refunded.
-
-On Twitter, a sizable portion of the community may be found, and it may be a good idea to involve it in your proposal during the deposit period to get them ready to vote.
-
-Once the deposit threshold is met, the Governance Depositor submits the xEIP to the governance contract, and the voting period starts.
+As explained in the previous paragraph, before a governance proposal enters the voting period (i.e., for a proposal to be voted on), the xEIP creator must deposit an amount of 10B MEX in the governance contract.
 
-Only when the proposal is vetoed, the deposit will be entirely burnt. Otherwise, deposits will not be burnt, they will be fully refunded, even if the vote did not pass, for example if the quorum or the "Yes" majority were not reached. During the voting period, users upvote proposal ("Yes" vote), downvote it ("No" vote), participate with a white vote which is neither for or against the proposal ("White" vote), or veto the proposal ("NoWithVeto" vote).
+Only when the proposal is vetoed, half of the deposit will be burnt, and the other half will be refunded. Otherwise, the deposit will not be burnt, and will be fully refunded, even if the vote did not pass, for example if the quorum or the "Yes" majority were not reached. During the voting period, users upvote proposal ("Yes" vote), downvote it ("No" vote), participate with a white vote which is neither for or against the proposal ("White" vote), or veto the proposal ("NoWithVeto" vote).
 
-As the xEIP creator, you should now keep tabs on the Energy holders who have and have not voted. To make sure they are aware of your plan, that they may ask you any questions about it, and that they are ready to vote, you should re-engage directly with the highest-ranking Energy holders.
+In a future version of the on-chain governance, in case you have not enough funds to cover the entire deposit, you will be allowed to ask friends and larger MEX holders to top up the remaining MEX tokens until it reaches the deposit threshold. This deposit threshold will have to be attained within a maximum of 10 days, otherwise all users will be refunded.
```

Comment:
"will be allowed to ask friends and larger MEX holders to top up the remaining MEX tokens until it reaches the deposit threshold" -> "will be allowed to associate with other MEX holders in order to provide minimal MEX deposit."